test/perf/CMakeLists.txt

```diff
@@ -1,5 +1,17 @@
 # SPDX-License-Identifier: Apache-2.0
 
+add_onnx_mlir_library(PerfLib
+  PerfHelper.cpp
+  EXCLUDE_FROM_OM_LIBS
+
+  INCLUDE_DIRS PRIVATE
+  ${ONNX_MLIR_SRC_ROOT}/third_party/benchmark/include
+  ${ONNX_MLIR_BIN_ROOT}/third_party/benchmark/include
+
+  LINK_LIBS PRIVATE
```

Comment:
Since PerfLib is meant to be linked by other test libs, I would make the dependencies (and probably the include dirs) PUBLIC. Then you won't have to add them below since PerfLib will carry them.

Reply:
got it, your input on this is appreciated as cmake magic is still eluding me :-)

Reply:
It looks like it's the shared libraries build that's failing. Let me try locally and see what the issue is. Probably a couple of hours to get the build all done and figured out, so I should have a fix this afternoon.